src/views/placeholder-product-downloads-view/index.tsx

```diff
@@ -37,7 +38,7 @@ const PlaceholderDownloadsView = (): ReactElement => {
   return (
     <SidebarSidecarLayout
       sidebarProps={{
-        menuItems: navData,
+        menuItems: navData as EnrichedNavItem[],
```

Comment:
This won't be the final output here, I just used casting to get the errors to go away so I could commit. 😊